src/Symfony/Component/Validator/Constraints/TypeValidator.php

```diff
@@ -20,6 +20,40 @@
  */
 class TypeValidator extends ConstraintValidator
 {
+    private static $validationFunctions = [
```

Comment:
You can easily replace static variable with array const like:
```php
const VALIDATOR_FUNCTIONS = [
        'bool' => 'is_bool',
        'boolean' => 'is_bool',
        'int' => 'is_int', // ...
];
```

Reply:
You can even define it as `private const`, as 7.1 is already even out of support phase officially.

Reply:
I agree. That's what I meant to do, actually 😄. Thank you!